modules/fluentd_new/elb.tf

```diff
@@ -0,0 +1,92 @@
+locals {
+  private_zone_id     = join("", aws_route53_zone.private.*.zone_id)
+  fluentd_server_port = 4224
+  fluentd_lb_port     = var.fluentd_port
+}
+
+##############################
+# Define an internal load balancer
+##############################
+
+# Internal Load balancer
+resource "aws_lb" "fluentd" {
```

Comment:
We are not enabling access logging for this LB:
1) For network load balancer, only TLS connections will be logged. However we didn't set up a TLS listener since we were originally using raw TCP forwarding. It is possible to add TLS encryption to Fluentd configs but that is outside the scope of this PR. In the current setup, no access logs will be logged to begin with, so no point enabling access logging.
2) Logs are already stored in S3 in addition to Elasticsearch, so storing additional access logs for the actual logs forwarded seems like an overkill.

Important ref for the future if we want to add TLS encryption and access logging: https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-access-logs.html